### DISCUSSION TITLE
Controlling f16 through joystick

### DISCUSSION THREAD

**Discussion by ajithmdh** [JSBSim-Team/jsbsim#487] [Q&A]
Hi,
I am new to JSBSim and Aerodynamics. Trying to control f16 using joystick. For banking I am setting fcs/aileron-cmd-norm and fcs/rudder-cmd-norm. 
It starts banking, but it does not stop banking even after I set aileron-cmd-norm and rudder-cmd-norm to zero.  What is that I am doing wrong here  ?
One more question is how to hold at some perticular bank angle ? Because even if I set fcs/aileron-cmd-norm and fcs/rudder-cmd-norm to a constant value, roll angle keep on increasing.

**Comment by DheerendraTomar** [comment]
Although I am not an expert in this field but from what I have learned is that, you will have to counter the bank angle first and then when it is near to zero, set the ailerons to zero.  Also, the banking is due to winds, you should use rudder constantly for that. 

For others, please correct me if I am wrong :D

**Reply by ajithmdh** [reply]
Thank you, countering the bank angle and then setting it to zero is working. But for it I am moving the joystick in opposite directon first and then I am bringing it to center.
For example if fcs/aileron-cmd-norm is set to 0.01, then for countering it I am giving values in opposite direction, say -0.08, then checking the roll, when it comes near zero I am setting ailerons and rudder to zero. Is this the same way it happens in actual flying also ? Because what I understand is if they will move control stick right, aircraft banks, when they bring it to center, it stops banking.
So should I take care countering the bank angle while bring back the control stick to center ?

**Reply by DheerendraTomar** [reply]
Read [FBW](https://www.aviationhunt.com/fly-by-wire/).

**Comment by seanmcleod** [comment]
What is you overall aim/goal? The reason I'm asking is to figure out whether the F-16 model is a good choice. The reason I mention that is because the F-16 model comes with a set of FBW command laws which sit in-between the pilot's commands from the joystick and the control surfaces with a control system.

So it's the not the same as for example a non-FBW aircraft like a C-172 where the pilot's inceptors (yoke, joystick, rudder pedals) are directly connected mechanically to the control surfaces.

Depending on what you're trying to achieve the FBW control system in-between could complicate your life.
